apps/src/bubbleChoice/types.ts

```diff
@@ -0,0 +1,9 @@
+// TODO: Ideally this type would only contain keys present in
+// translated string JSON files (ex. apps/i18n/bubbleChoice/en_us.json).
+// However, this requires depending on files outside of apps/src,
+// so this approach is still being investigated. For now, this type
+// is an object whose keys are all functions which return strings,
+// matching what we expect for a locale object.
+export type BubbleChoiceLocale = {
```

Comment:
We can use the new locale exporting system that uses the JSON file directly now (I think we need to update this in StandaloneVideo too). See `apps/src/music/locale.ts`. That being said if we don't have any locale strings for this app type yet, we don't even need this type or the `locale` file yet.

Reply:
Ah, I did borrow this from `StandaloneVideo`.  That said, we don't have any app-specific locale strings yet, so I will remove these files for now.